app/src/i18n/locales/de/dashboard.json

```diff
@@ -0,0 +1,133 @@
+  "stationary-energy-description": "Dieser Sektor befasst sich mit Emissionen, die durch die Erzeugung von Elektrizität, Wärme und Dampf sowie deren Verbrauch entstehen.",
+  "add-data-to-sector": "Daten hinzufügen",
+  "scope-required-for-gpc": "Erforderlicher Umfang für GPC-Basisinventar",
+  "scope-required-for-gpc+": "Erforderlicher Umfang für GPC Basic+ Inventar",
+  "sub-sectors-required": "Erforderlicher Teilsektor",
+  "view-more": "MEHR ANSEHEN",
+  "view-less": "WENIGER ANZEIGEN",
+  "transportation": "Verkehr",
+  "transportation-description": "Dieser Sektor befasst sich mit den Emissionen aus dem Transport von Waren und Personen innerhalb der Stadtgrenze.",
+  "waste": "Abfall und Abwasser",
+  "waste-description": "Dieser Sektor umfasst Emissionen, die aus Abfallmanagementprozessen erzeugt werden.",
+  "ippu": "Industrielle Prozesse und Produktnutzung (IPPU)",
+  "ippu-description": "Dieser Sektor umfasst die Treibhausgasemissionen aus industriellen Prozessen, die Materialien umwandeln, wie beispielsweise bei der Stahlproduktion und chemischen Herstellung.",
+  "afolu": "Landwirtschaft, Forstwirtschaft und Landnutzung (AFOLU)",
+  "afolu-description": "Dieser Sektor umfasst Emissionen aus Landwirtschaft, Forstwirtschaft und Landnutzungsänderungen, einschließlich Viehzucht, Rodung und Aktivitäten wie Düngemittelanwendung und Reisanbau.",
+  "unnamed-sector": "Unbenannter Sektor",
+  "try-again": "Versuchen Sie es erneut",
+  "preparing-dataset": "Vorbereitung Ihres Datensatzes zum Herunterladen",
+  "wait-fetch-data": "Bitte warten Sie, während wir Ihre Daten abrufen",
+  "download-complete": "Inventarbericht Download abgeschlossen!",
+  "downloading-data": "Ihre Daten herunterladen",
+  "download-failed": "Download fehlgeschlagen",
+  "download-error": "Es gab einen Fehler beim Herunterladen",
+  "tab-emission-inventory-calculation-title": "Inventarberechnung",
+  "emission-inventory-calculation-title": "Berechnung des Emissionsinventars",
+  "tab-emission-inventory-results-title": "Ergebnisse des Emissionsinventars",
+  "inventory-format-basic": "Inventarformat: GPC BASIC",
+  "inventory-format-basic+": "Inventarformat: GPC BASIC+",
+  "view-progress-in-each-sector": "Sehen Sie den Fortschritt der Datenerfassung in jedem vom GPC erforderlichen Sektor.",
+  "sector-emissions": "Sektor-Emissionen",
+  "sector-data": "Sektordaten",
+  "total-emissions-tooltip": "Hier können Sie die Gesamtemissionen im Jahr {{year}} sehen, wenn das Inventar abgeschlossen ist.",
+  "year": "Jahr",
+  "population-year": "Jahr: {{year}}",
+  "total-emissions": "Gesamtemissionen",
+  "total-ghg-emissions-in-year": "Gesamte Treibhausgasemissionen im Jahr {{year}}",
+  "emissions-per-capita-in-year": "Emissionen pro Kopf im Jahr {{year}}",
+  "%-of-country's-emissions": "% der Emissionen des Landes",
+  "top-emissions": "Höchste Emissionen",
+  "ghg-by-sector-heading": "Geschichte der Treibhausgasemissionen nach Sektoren",
```

Comment:
I assume this talks about the history as time series data. The translation here is literally history.
Better "Verlauf der Treibhausgasemissionen nach Sektoren"